### ISSUE TITLE
Crash [1.12]

### ISSUE THREAD

**Issue by Intipablo** [codetaylor/dropt#4]
Started crashing after adding a .json file to the config folder (Already had one inside that worked).

Crash Log:
https://pastebin.com/1FYRgtxh

flint.json (I added this one and it started to crash):
https://pastebin.com/dcRajdSM

sticks.json (this was the first one I added and it worked fine):
https://pastebin.com/yNUrdse4

**Comment by codetaylor** [comment]
```
[03:48:56] [Server thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_151]
	at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_151]
	at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:721) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:666) [MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:185) [chd.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:524) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_151]
Caused by: java.lang.NullPointerException
	at com.codetaylor.mc.dropt.modules.dropt.rule.drop.DropModifier.modifyDrops(DropModifier.java:119) ~[DropModifier.class:?]
	at com.codetaylor.mc.dropt.modules.dropt.events.EventHandler.onHarvestDropsEvent(EventHandler.java:99) ~[EventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_224_EventHandler_onHarvestDropsEvent_HarvestDropsEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:179) ~[EventBus.class:?]
	at net.minecraftforge.event.ForgeEventFactory.fireBlockHarvesting(ForgeEventFactory.java:316) ~[ForgeEventFactory.class:?]
	at net.minecraft.block.Block.func_180653_a(Block.java:572) ~[aow.class:?]
	at net.minecraft.block.Block.func_176226_b(Block.java:564) ~[aow.class:?]
	at net.minecraft.block.Block.func_180657_a(Block.java:735) ~[aow.class:?]
	at net.minecraft.server.management.PlayerInteractionManager.func_180237_b(PlayerInteractionManager.java:327) ~[or.class:?]
	at net.minecraft.server.management.PlayerInteractionManager.func_180785_a(PlayerInteractionManager.java:244) ~[or.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147345_a(NetHandlerPlayServer.java:702) ~[pa.class:?]
	at net.minecraft.network.play.client.CPacketPlayerDigging.func_148833_a(SourceFile:40) ~[lp.class:?]
	at net.minecraft.network.play.client.CPacketPlayerDigging.func_148833_a(SourceFile:10) ~[lp.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[hv$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_151]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:1.8.0_151]
	at net.minecraft.util.Util.func_181617_a(SourceFile:46) ~[h.class:?]
```

**Comment by codetaylor** [comment]
Fixed in version 1.12.2-1.4.4

Thanks for finding and reporting this! :)

**Comment by Intipablo** [comment]
All good, thanks for the quick fix!
